tools/js-sdk-release-tools/src/llc/generateRLCInPipeline/generateRLCInPipeline.ts

```diff
@@ -243,7 +243,7 @@ export async function generateRLCInPipeline(options: {
         logger.info(`Start to run command 'node common/scripts/install-run-rush.js pack --to ${packageName} --verbose'.`);
         execSync(`node common/scripts/install-run-rush.js pack --to ${packageName} --verbose`, {stdio: 'inherit'});
         if (!options.skipGeneration) {
-            const changelog = await generateChangelogAndBumpVersion(packagePath);
+            const changelog = await generateChangelogAndBumpVersion(relativePackagePath);
```

Comment:
serveral prs with same change, which one should be reviewed/approved? not sure we could have a test case covered.

Reply:
> serveral prs with same change, which one should be reviewed/approved
it's due to pipeline failed to find pr, I'm not sure why, maybe it's a github bug, after a while, everything became normal
please review/approve this one
> not sure we could have a test case covered.
forget to add test result: https://github.com/test-repo-billy/azure-rest-api-specs/pull/3557
this caused by manual test by me. generateChangelogAndBumpVersion always use relative package path